Library/Formula/xalan-c.rb

```diff
@@ -11,8 +11,13 @@ class XalanC < Formula
     sha256 "fcfe6027b7d366f6a2bff783e0ab1e9abfc7c38c1a6fd31fa4a2fb9d325a2819" => :mavericks
   end
 
+  option "with-docs", "Install HTML docs"
   option :universal
 
+  if build.with? "docs"
+    depends_on "doxygen" => :build if build.with? "docs"
```

Comment:
This is now fixed.  Sorry for the delay; I must have missed the notification of your comment.